share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,35 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    chompContents: true,
```

Comment:
It looks like `condense` is no longer a block helper. [The example from the docs](http://docs.duckduckhack.com/frontend-reference/handlebars-helpers.html#condense) has it being used like so:
```handlebars
{{condense myString maxlen="135" truncation="..."}}
```